### DISCUSSION TITLE
建议增加赞粒攻略任务：坚持赞保障金任务签到

### DISCUSSION THREAD

**Discussion by forkWo** [constanline/XQuickEnergy#498] [Ideas/我也有一个梦想]
![Screenshot_2024-02-07-12-01-27-026_com.eg.android.AlipayGphone.jpg](https://github.com/constanline/XQuickEnergy/assets/119728345/5cb458db-db18-4df2-9680-af41136f32ab)

**Comment by xzcgg** [comment]
期待大佬增加这个功能
